src/BuildIntegration/Microsoft.NETCore.Native.Unix.props

```diff
@@ -49,6 +49,7 @@ See the LICENSE file in the project root for more information.
     <LinkerArg Include="@(NativeLibrary)" />
     <LinkerArg Include="@(AdditionalNativeLibrary)" />
     <LinkerArg Include="-g" />
+    <LinkerArg Include="-Wl,-rpath,'$ORIGIN'" />
```

Comment:
According to https://linux.die.net/man/8/ld-linux "-Wl,-rpath,'$ORIGIN'" is the right syntax.